library/src/main/java/org/apache/apex/malhar/lib/dedup/BoundedDedupOperator.java

```diff
@@ -156,8 +156,7 @@ protected Slice getKey(Object tuple)
   protected void putManagedState(Object tuple)
   {
     Slice key = getKey(tuple);
-    ((ManagedTimeStateImpl)managedState).put(getBucketId(key), DEFAULT_CONSTANT_TIME,
-        key, new Slice(new byte[0]));
+    ((ManagedTimeStateImpl)managedState).put(getBucketId(key), DEFAULT_CONSTANT_TIME, key, new Slice(null, 0, 0));
```

Comment:
I guess I am not able to put forward my point correctly. 
Below is the trace for the failing test which indicates that the Slice created above is causing the test to fail.
```
java.lang.IllegalArgumentException: Invalid slice: offset=0, length=0 array.length=0
	at com.datatorrent.netlet.util.Slice.<init>(Slice.java:43)
	at org.apache.apex.malhar.lib.utils.serde.BufferSlice.<init>(BufferSlice.java:48)
	at org.apache.apex.malhar.lib.utils.serde.BufferSlice.<init>(BufferSlice.java:58)
	at org.apache.apex.malhar.lib.utils.serde.SliceUtils.toBufferSlice(SliceUtils.java:111)
	at org.apache.apex.malhar.lib.state.managed.Bucket$DefaultBucket.put(Bucket.java:421)
	at org.apache.apex.malhar.lib.state.managed.AbstractManagedStateImpl.putInBucket(AbstractManagedStateImpl.java:286)
	at org.apache.apex.malhar.lib.state.managed.ManagedTimeUnifiedStateImpl.put(ManagedTimeUnifiedStateImpl.java:72)
	at org.apache.apex.malhar.lib.dedup.TimeBasedDedupOperator.putManagedState(TimeBasedDedupOperator.java:189)
	at org.apache.apex.malhar.lib.dedup.AbstractDeduper.processAuxiliary(AbstractDeduper.java:316)
	at org.apache.apex.malhar.lib.dedup.AbstractDeduper.handleIdleTime(AbstractDeduper.java:293)
	at org.apache.apex.malhar.lib.dedup.DeduperTimeBasedPOJOImplTest.testDedup(DeduperTimeBasedPOJOImplTest.java:98)
```